docs/api/turn-by-turn/api-reference.md

```diff
@@ -122,6 +122,7 @@ The following options are available for `auto`, `bus`, `hov`, `taxi`, and `truck
 | :-------------------------- | :----------- |
 | `height` | The height of the vehicle (in meters). |
 | `width` | The width of the vehicle (in meters). |
+| `exclude_unpaved` | This value indicates the willingness to take unpaved roads. If `exclude_unpaved` is set to 1, unpaved roads are excluded from the process of generating the route path, otherwise included. |
```

Comment:
i think clearer language around this option being binary would be helpful

```suggestion
| `exclude_unpaved` | This value indicates the whether or not the path should include unpaved roads. If `exclude_unpaved` is set to 1, unpaved roads not allowed to be used in the process of generating the route path, otherwise they are allowed. |
```